.circleci/config.yml

```diff
@@ -261,9 +261,19 @@ jobs:
       - run: make coverage-report-html
       - store_artifacts: *store_cover_db
       - run: make coverage-report-codecov
+      - run:
+          # https://github.com/codecov/codecov-circleci-orb/issues/157
+          name: codecov gpg workaround for gpg 2.4 as included in Leap 15.6
+          command: mkdir -p ~/.gnupg
       - codecov/upload:
           file: cover_db/codecov.json
-          xtra_args: -v -Z
+          cli_args: -v
+          # Workaround broken uploader validation after upgrade to Leap 15.6
+          # base
+          # See https://github.com/os-autoinst/openQA/pull/5632
+          # and
+          # https://progress.opensuse.org/issues/157984
+          validate: false
```

Comment:
Not sure if you already tried that (in that case please disregard this suggestion), but if the linked comment is not wrong, we might be able to keep the validation with the the `mkdir` workaround.

```suggestion
```

Reply:
of course, that was my idea with calling this PR "fixed". Got confused here as well :/ Thx.